capstone-frontend/src/components/UserCard.jsx

```diff
@@ -12,7 +12,7 @@ export class UserCard extends Component {
     }
     fetch(`/api/user?id=${window.localStorage.getItem('userID')}`)
       .then(response => response.json())
-      .then(user => this.setState({isFriend:user.friendIDs.includes(this.props.user.id)}))
+      .then(user => this.setState({isFriend: user.friendIDs && user.friendIDs.includes(this.props.user.id)}))
```

Comment:
If user.friendIDs doesn't exist, does this statement mean that the state is automatically false?

Reply:
Yes it does. It also logically works, as if they have no friends, then this user is definitely not their friend.